plugins/legacy-plugin-chart-calendar/src/controlPanel.ts

```diff
@@ -16,15 +16,16 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import { t, legacyValidateInteger } from '@superset-ui/core';
 import {
-  t,
-  legacyValidateInteger,
-  D3_TIME_FORMAT_OPTIONS,
```

Comment:
This import was referencing `@superset-ui/core`, but it's actually an export in `@superset/ui/chart-controls`, aka the control didn't work (only showing default value): 
![image](https://user-images.githubusercontent.com/33317356/102778961-2767b800-439c-11eb-9337-500ab2b020af.png)